crates/nu-cli/src/commands/parse/command.rs

```diff
@@ -41,6 +41,19 @@ impl WholeStreamCommand for Command {
     ) -> Result<OutputStream, ShellError> {
         operate(args, registry).await
     }
+
+    fn examples(&self) -> Vec<Example> {
+        let mut row = IndexMap::new();
+        row.insert("foo".to_string(), Value::from("hi"));
+        row.insert("bar".to_string(), Value::from("there"));
+        let mut result_table = Vec::new();
+        result_table.push(UntaggedValue::row(row));
```

Comment:
The last error is an expectation for tagged values in this vec.

```rust
result_table.push(UntaggedValue::row(row).into_untagged_value());
```

All of this highlights that having some helpers to create tables and rows would be useful 🙂